### DISCUSSION TITLE
How are you using `handle` export in your remix app?

### DISCUSSION THREAD

**Discussion by cliffordfajardo** [remix-run/remix#7560] [Q&A]
### How are you using the remix handle API in your remix app?
Curious to hear creative use cases or more content about this in general so created this discussion 😄 

Remix docs has one example showcasing how you can build dynamic breadcrumbs; 
https://remix.run/docs/en/main/guides/breadcrumbs

<img src="https://github.com/remix-run/remix/assets/6743796/43e6f313-8f91-42fb-b512-9162e2049762" height="350" />

## References
- [Remix Discord Thread on `handle` export use cases](https://discord.com/channels/770287896669978684/1157687817330237583)
- [Remix Handle docs](https://discord.com/channels/770287896669978684/1157687817330237583)

**Comment by kentcdodds** [comment]
I'm using it for breadcrumbs and sitemaps here:

https://github.com/epicweb-dev/epic-stack/blob/main/app/routes/settings%2B/profile.two-factor.disable.tsx#L14

https://github.com/epicweb-dev/epic-stack/blob/main/app/routes/settings%2B/profile.tsx#L37

https://github.com/epicweb-dev/epic-stack/blob/main/app/routes/_seo%2B/sitemap%5B.%5Dxml.ts

**Comment by kiliman** [comment]
I've used it to override a `<Layout>` for one-off routes.

**Comment by AhmedNawaf** [comment]
I'm using it in a multi-step registration form to track the current step.
Where each step is a child route exporting a handle:
```typescript
export const handle = {
  step: 3, // or any number
};
```
